cmd/image-autobumper/README.md

```diff
@@ -0,0 +1,33 @@
+# Image Autobumper
+
+Image Autobumper is a tool for automatically updating the version of a Docker image in a GitHub repository. It is designed to automatically detected the latest version of the image and update the version in the repository based on the provided configuration.
+
+Key features:
+* Automatically detects the latest version of the image
+* Automatically detects places in the file where image urls are defined
+* Automatically updates the version of the image in the files
+
+## Quickstart Guide
+
+You can use Image Autobumper in your GitHub workflow to update the version of a Docker image in a GitHub repository. It autoamtically detects yaml files (both `.yaml` and `.yml`) in the repository. You can configure the tool to update the version of the image in other files with `extraFiles` option in the configuration file.
```

Comment:
This is mainly repetition of what you've already said.

Reply:
Removed repetetition, at least tried to remove all of them.